clientconn.go

```diff
@@ -1681,14 +1680,14 @@ func (cc *ClientConn) connectionError() error {
 //
 // Doesn't grab cc.mu as this method is expected to be called only at Dial time.
 func (cc *ClientConn) parseTargetAndFindResolver() error {
-	channelz.Infof(logger, cc.channelz, "original dial target is: %q", cc.target)
+	logger.Infof("original dial target is: %q", cc.target)
 
 	var rb resolver.Builder
 	parsedTarget, err := parseTarget(cc.target)
 	if err != nil {
-		channelz.Infof(logger, cc.channelz, "dial target %q parse failed: %v", cc.target, err)
+		logger.Infof("dial target %q parse failed: %v", cc.target, err)
```

Comment:
- This doesn't seem worth keeping.  We're going to try some more stuff that might make it valid, and something as simple as `hostname:port` will trigger this.

Reply:
Deleted (and switched conditional to err == nil.